groundswell k8s and docker-compose instructions (#376)
seqeralabs/nf-tower-docs: docs/enterprise/kubernetes.md

## Optional addons

### Groundswell

> Groundswell is a utility that optimizes pipelines and performs resource prediction, and can be installed in your Kubernetes cluster with the following manifest:

Reviewer: ```suggestion
Groundswell is a service that optimizes the resource requests of pipelines. It can be installed in your Kubernetes cluster with the following manifest:
```
